cl/citations/management/commands/cl_add_parallel_citations.py

```diff
@@ -218,7 +218,11 @@ def add_groups_to_network(self, citation_groups):
         for group in citation_groups:
             edge_list = make_edge_list(group)
             for edge in edge_list:
-                if any(e for e in edge if e.reporter_found in ["Id.", "Cr."]):
+                if any(
+                    e
+                    for e in edge
+                    if e.groups.reporter_found in ["Id.", "Cr."]
+                ):
```

Comment:
I usually let `black` do its thing, but this is kind of nasty. Want to try to make it not so terrible?

Reply:
Refactored in cc4f608